src/ddmd/statement.d

```diff
@@ -1341,6 +1450,66 @@ extern (C++) final class ConditionalStatement : Statement
 }
 
 /***********************************************************
+ * Static foreach statements, like:
+ *      void main(){
+ *           static foreach(i; 0 .. 10)
+ *           {
+ *               pragma(msg, i);
+ *           }
+ *      }
+ */
+extern (C++) final class StaticForeachStatement : Statement
+{
+    StaticForeach sfe;
+
+    bool cached = false;
```

Comment:
It seems not to be necessary. I'll remove it. (I think it was just defensive coding, as it was necessary for static foreach declarations.)